### PR TITLE
Add icon for Circumvention test group

### LINKED ISSUE #72: Add circumvention tool tests icon
@holantonela mentioned there is an icon for the circumvention tests already in: https://github.com/ooni/probe/issues/804#issuecomment-559800723

We should add the icon and the color to the design-system.

The icon needs to be added in SVG form in here: https://github.com/ooni/design-system/tree/master/components/svgs/icons.

### PR DESCRIPTION
**Pull request by sarathms**
Icon added for Circumvention test group. I used the SVG @holantonela gave [here](https://github.com/ooni/design-system/issues/72#issuecomment-559809954).

Fixes #72 

image

### CODE REVIEW

File: components/icons/index.js [hunk 1]
```diff
@@ -37,4 +37,5 @@ export { default as NettestGroupWebsites } from './NettestGroupWebsites'
 export { default as NettestTelegram } from './NettestTelegram'
 export { default as NettestWhatsApp } from './NettestWhatsApp'
 export { default as NettestVanillaTor } from './NettestVanillaTor'
+export { default as NettestGroupCircumvention } from './NettestGroupCircumvention'
```
Comment (hellais) [inline components/icons/index.js]: Maybe this should be placed above together with the other `NettestGroup` components?

File: components/icons/NettestGroupCircumvention.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Icon from 'react-icon-base'
+
+const NettestGroupCircumvention = props => (
```
Comment (hellais) [inline components/icons/NettestGroupCircumvention.js]: This icon should be generated using the npm command `create-icons`: https://github.com/ooni/design-system/blob/master/package.json#L59.

The svg of the icon should instead live inside of https://github.com/ooni/design-system/tree/master/components/svgs/icons

Comment (sarathms) [reply components/icons/NettestGroupCircumvention.js]: Generated with the script now.

**Comment by sarathms** [comment]
@hellais Can you share an appropriate svg for Vanilla Tor test? The one is used in #61 wasn't generated with the script and it doesn't seem to look right when I use that svg now.

**Comment by hellais** [comment]
@sarathms I pushed it directly to this branch.

I also added the icons to the storybook so we can preview them easier.

Here is how they look like:

Screenshot 2019-12-05 at 12 45 13

To me they look OK, but it's probably good if @holantonela also checks the Tor icon and let's us know if it looks good.
If it doesn't we can update it as part of another PR or issue.

**Pull request merged**